tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -1007,6 +1007,239 @@ void test_multibyte_format_strings() {
+    throw_helper(STR("{0:-"), 'c');
+    throw_helper(STR("{0:-}"), 'c');
+    throw_helper(STR("{0:-}"), STR("abc"));
+    throw_helper(STR("{0:-}"), reinterpret_cast<void*>(0x42));
+}
+
+template <class charT>
+void libfmt_formatter_test_space_sign() {
+    assert(format(STR("{0: }"), 42) == STR(" 42"));
+    assert(format(STR("{0: }"), -42) == STR("-42"));
+    assert(format(STR("{0: }"), 42) == STR(" 42"));
+    // behavior differs from libfmt, but conforms
+    assert(format(STR("{0: }"), 42u) == STR(" 42"));
+    assert(format(STR("{0: }"), 42l) == STR(" 42"));
+    // behavior differs from libfmt, but conforms
+    assert(format(STR("{0: }"), 42ul) == STR(" 42"));
+    assert(format(STR("{0: }"), 42ll) == STR(" 42"));
+    // behavior differs from libfmt, but conforms
+    assert(format(STR("{0: }"), 42ull) == STR(" 42"));
+    assert(format(STR("{0: }"), 42.0) == STR(" 42"));
+    assert(format(STR("{0: }"), 42.0l) == STR(" 42"));
+    throw_helper(STR("{0: "), 'c');
+    throw_helper(STR("{0: }"), 'c');
+    throw_helper(STR("{0: }"), STR("abc"));
+    throw_helper(STR("{0: }"), reinterpret_cast<void*>(0x42));
+}
+
+template <class charT>
+void libfmt_formatter_test_hash_flag() {
+    assert(format(STR("{0:#}"), 42) == STR("42"));
+    assert(format(STR("{0:#}"), -42) == STR("-42"));
+    assert(format(STR("{0:#b}"), 42) == STR("0b101010"));
+    assert(format(STR("{0:#B}"), 42) == STR("0B101010"));
+    assert(format(STR("{0:#b}"), -42) == STR("-0b101010"));
+    assert(format(STR("{0:#x}"), 0x42) == STR("0x42"));
+    assert(format(STR("{0:#X}"), 0x42) == STR("0X42"));
+    assert(format(STR("{0:#x}"), -0x42) == STR("-0x42"));
+    assert(format(STR("{0:#o}"), 0) == STR("0"));
+    assert(format(STR("{0:#o}"), 042) == STR("042"));
+    assert(format(STR("{0:#o}"), -042) == STR("-042"));
+    assert(format(STR("{0:#}"), 42u) == STR("42"));
+    assert(format(STR("{0:#x}"), 0x42u) == STR("0x42"));
+    assert(format(STR("{0:#o}"), 042u) == STR("042"));
+
+    assert(format(STR("{0:#}"), -42l) == STR("-42"));
+    assert(format(STR("{0:#x}"), 0x42l) == STR("0x42"));
+    assert(format(STR("{0:#x}"), -0x42l) == STR("-0x42"));
+    assert(format(STR("{0:#o}"), 042l) == STR("042"));
+    assert(format(STR("{0:#o}"), -042l) == STR("-042"));
+    assert(format(STR("{0:#}"), 42ul) == STR("42"));
+    assert(format(STR("{0:#x}"), 0x42ul) == STR("0x42"));
+    assert(format(STR("{0:#o}"), 042ul) == STR("042"));
+
+    assert(format(STR("{0:#}"), -42ll) == STR("-42"));
+    assert(format(STR("{0:#x}"), 0x42ll) == STR("0x42"));
+    assert(format(STR("{0:#x}"), -0x42ll) == STR("-0x42"));
+    assert(format(STR("{0:#o}"), 042ll) == STR("042"));
+    assert(format(STR("{0:#o}"), -042ll) == STR("-042"));
+    assert(format(STR("{0:#}"), 42ull) == STR("42"));
+    assert(format(STR("{0:#x}"), 0x42ull) == STR("0x42"));
+    assert(format(STR("{0:#o}"), 042ull) == STR("042"));
+
+    // differs from libfmt, but conforms

```

Comment:
Ultra-nitpick, the other comments said `// behavior differs from libfmt, but conforms` and if these matched it'd be easier to search for all occurrences. Occurs twice below in this function.